packages/remote-feature-flag-controller/src/client-config-api-service/client-config-api-service.test.ts

```diff
@@ -0,0 +1,311 @@
+import type { FeatureFlags } from '../remote-feature-flag-controller-types';
+import {
+  ClientType,
+  DistributionType,
+  EnvironmentType,
+} from '../remote-feature-flag-controller-types';
+import { ClientConfigApiService } from './client-config-api-service';
+
+const BASE_URL = 'https://client-config.api.cx.metamask.io/v1';
+
+// eslint-disable-next-line jest/prefer-spy-on
+console.error = jest.fn();
```

Comment:
This is better, actually i looked through tests for other controllers and found this pattern and i thought it would be the one we are following, for example: https://github.com/MetaMask/core/blob/main/packages/transaction-controller/src/helpers/IncomingTransactionHelper.test.ts#L19. 
=> Maybe we can create a refactor ticket to remove this pattern and use spy from jest to mute the error?
=> the modification is captured in 66ebdbeb6f0108b46c47df2338123cb4d087799b

Reply:
Ah, makes sense. Yes, I can make a new ticket for that.